Sources/TuistGenerator/Generator/ConfigGenerator.swift

```diff
@@ -276,11 +276,21 @@ final class ConfigGenerator: ConfigGenerating {
         var settings: SettingsDictionary = [:]
         settings["TEST_TARGET_NAME"] = .string("\(app.target.name)")
         if target.product == .unitTests {
-            settings["TEST_HOST"] =
-                .string(
-                    "$(BUILT_PRODUCTS_DIR)/\(app.target.productNameWithExtension)/$(BUNDLE_EXECUTABLE_FOLDER_PATH)/\(app.target.productName)"
+            do {
+                let path = try AbsolutePath(validating: "$(BUILT_PRODUCTS_DIR)")
```

Comment:
this fails as it's not a valid path :/

Reply:
added now a sed post script modification
# BUNDLE_EXECUTABLE_FOLDER_PATH// incompatibility in tuist >= 3.22.0
    sed -i -e ‘s/BUNDLE_EXECUTABLE_FOLDER_PATH)\//BUNDLE_EXECUTABLE_FOLDER_PATH)/g’ “XXX.xcodeproj/project.pbxproj”